launch/zed.launch

```diff
@@ -0,0 +1,16 @@
+<launch>
+    <!-- Odometry coordinate frame -->
+    <arg name="odometry_frame"       default="map" />
+    <arg name="camera_model"          default="0" /> <!-- 0=ZED, 1=ZEDM-->
+    <arg name="serial_number"           default="0" /> 
+
+    <group ns="zed">
+        <include file="$(find zed_wrapper)/launch/zed_camera.launch">
```

Comment:
should this be `zed_ros_wrapper` instead of `zed_wrapper`.

Reply:
zed_wrapper is right https://github.com/stereolabs/zed-ros-wrapper/blob/master/package.xml#L3

Reply:
did you test it ?

Reply:
Yeah. You should try it though